### PR TITLE
Appveyor

### PR DESCRIPTION
**Pull request by ciechowoj**
A quick fix to restore x86_64 AppVeyor tests.

### CODE REVIEW

File: unit_tests/UnitTests.d [hunk 1]
```diff
@@ -880,7 +880,7 @@ void foo ();
 D", options);
 
 }
-
+/+
```
Comment (jacob-carlborg) [inline unit_tests/UnitTests.d]: Can this test still be enabled for the other platforms?

**Comment by jacob-carlborg** [comment]
I took the library to address the comment myself. I amended the last commit and force pushed.

**Comment by ciechowoj** [comment]
> I took the *liberty* to address the comment myself. I amended the last commit and force pushed.

Thank you.

**Comment by jacob-carlborg** [comment]
Thanks.

**Pull request merged**